test/e2e/playwright/tests/desktop/identifier_first/oidc.login.spec.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2024 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+import { faker } from "@faker-js/faker"
+import { expect } from "@playwright/test"
+import { getSession } from "../../../actions/session"
+import { test } from "../../../fixtures"
+import { toConfig } from "../../../lib/helper"
+
+for (const mitigateEnumeration of [true, false]) {
+  test.describe(`account enumeration protection ${
+    mitigateEnumeration ? "on" : "off"
+  }`, () => {
+    test.use({
+      configOverride: toConfig({ mitigateEnumeration }),
+    })
+
+    test("login", async ({ page, config, kratosPublicURL }) => {
```

Comment:
OIDC also works for two step:

1. Enter email
2. Account with email is found that has OIDC set up
3. Show only available oidc providers if account enumeration protection if off
3. Show all available oidc providers if account enumeration protectionif on

I think we should cover that case here too

Reply:
ah good, point. Will add.

Reply:
Done.